src/test/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandlerTest.java

```diff
@@ -1979,87 +1964,46 @@ public void getStatisticsSlowDown(TestContext context) throws Exception {
 
         // Test increasing slowDown Time within 5 seconds
         eventBusSend(buildEnqueueOperation("stat_b", "item_b_1"), handler -> {
-            ArrayList<HashMap<String, Object>> response = when().get("/queuing/statistics")
-                .then()
-                .assertThat().statusCode(200)
-                .extract().path("queues");
-            context.assertTrue(response.size() == 1);
-            context.assertTrue(response.get(0).get("name").equals("stat_b"));
-            context.assertTrue(response.get(0).get("size").equals(1));
-            context.assertTrue(((int) response.get(0).get("failures")) >= 1);
-            context.assertTrue(((int) response.get(0).get("slowdownTime")) >= 0);
-
-            // now we wait just some time up until the failures and slowdowns are applied
```

Comment:
It looks like this was accidentally removed?

Reply:
The checks are now part of the assertQueueState method itself and therefore not needed anymore.  As we now do wait up until the requested state is reached, we could check for particular specific foreseen situation.

Reply:
Hmm yes, but the 5 sec sleep on line #1993 is gone now but still mentioned on line #1980 (left)/ #1965 (right) ?

Reply:
Aha, that's what you mean.  The strategy changed but the 5 seconds are still valid as this is now the maximum time we wait up until the expected situation must be fullfilled.  There is no explicit wait anymore but a constantly check during this time.

Reply:
OK, thx for the clarification